nym-wallet/src/components/Delegation/DelegateModal.tsx

```diff
@@ -134,7 +132,7 @@ export const DelegateModal: React.FC<{
     }
   };
 
-  const handleConfirm = async ({ mixId, value }: { mixId: number; value: DecCoin }) => {
+  const handleConfirm = async ({ id, value }: { id: number; value: DecCoin }) => {
```

Comment:
I think using `mixId` was more explicit here (I attempted to use it everywhere on rust side to make it clear what we're actually referring to)

Reply:
Yes I understand, but there is already a variable declared as `mixId` in the upper scope, so ESLint complained about that, and it's true that it's not a good practice to have overlaps of variable name from different scopes, in a TS code base.
So this change fix that lint error. In addition, reading the function body, it's pretty clear for what `id` actually stand for.

Reply:
But I can find a compromise like something:
```
async ({ mixId: id, value }: { mixId: number; value: DecCoin }) => {
```